services/web/src/app/globals.css

```
@@ -2,6 +2,7 @@
 
 @plugin "daisyui" {
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@plugin" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)